README.md

```diff
@@ -103,6 +103,7 @@ This file contains a number of front-end interview questions that can be used wh
 * Have you played around with the new CSS Flexbox or Grid specs?
 * How is responsive design different from adaptive design?
 * Have you ever worked with retina graphics? If so, when and what techniques did you use?
+* Which one is better translate() vs Position:Abs;Top/left?
```

Comment:
I have used **better** because it is like asking the candidate like there are three options do you feel which option is better compared to other. According to me better word in this case choosing right one out of some options. If you have something different words please suggest me.